app/code/Codifi/Sales/view/adminhtml/templates/order/create/create.phtml

```diff
@@ -0,0 +1,37 @@
+<?php
+/**
+ * Codifi_Sales
+ *
+ * @copyright   Copyright (c) 2021 Codifi
+ * @author      Pavel Zelenevich <pzelenevich@codifi.me>
+ */
+
+/**
+ * @var \Magento\Framework\View\Element\Template $block
+ */
+$viewModel = $block->getOrderCreateViewModel();
+$attributeValue = $viewModel->getOptionsOrderType();
+?>
+<fieldset class="fieldset admin__fieldset " id="order_type_fieldset">
+    <div class="admin__field field field-order_type required _required">
+        <label class="label admin__field-label" for="order_type_select">
+            <span><?= $block->escapeHtml(__('Order Type')) ?></span>
+        </label>
+        <div class="admin__field-control control">
+            <select id="order_type_select" name="order[order_type]" class="required-entry _required select admin__control-select">
+                <?php foreach ($attributeValue as $item): ?>
+                <option value="<?= $block->escapeHtml(__($item['value'])) ?>">
```

Comment:
You should also specified `selected` attribute for appropriate restore of the field value on reorder